superset-frontend/src/SqlLab/components/SqlEditorLeftBar/index.tsx

```diff
@@ -92,6 +93,7 @@ export default function SqlEditorLeftBar({
   // Ref needed to avoid infinite rerenders on handlers
   // that require and modify the queryEditor
   const queryEditorRef = useRef<QueryEditor>(queryEditor);
+  const [noDbFoundTitle, setTitleForNoDBFound] = useState(false);
```

Comment:
can we change this to `isDBConnected` instead?

Reply:
Yeah, the name `noDbFoundTitle` makes me thing this is a string, not a boolean.

Reply:
Yep